### DISCUSSION TITLE
Use existing caddy setup

### DISCUSSION THREAD

**Discussion by gymnae** [gravitl/netmaker#1447] [Q&A] [answered]
The most up-to-date version of the docker-compose file seems to want to install traefik.
I have a pretty well running caddy setup and would like to add netmaker as an additional service on my server. 

Thus two questions:

1. Why is caddy apparently completely removed nowß
2. Can I integrate netmaker into an existing caddy landscape?

**Comment by mattkasun** [comment, marked as answer]
Traefik permits the proxying of mqtts traffic and therefore with traefik a netmaker server can be setup that only exposes tcp 443 (and wireguard udp ports) to the internet.

Netmaker can still be used with Caddy as the reverse proxy.

**Comment by gymnae** [comment]
I used a compose file from an older release as a base and now have it installed and running.
I needed to create a DNS entry for broker.mydomain.tld and open firewall port 8883.
